static/bidder-info/resetdigital.yaml

```diff
@@ -0,0 +1,19 @@
+    endpoint: http://b-us-east14.resetdigital.co:9001
+    meta-info:
+      maintainer-email: biddersupport@resetdigital.co
+      app-media-types:
+        - banner
+        - video
+        - audio
```

Comment:
Adapter code does not read any value from `request.App` .   @bruno-siira should remove `app-media-types` if adapter does not support  app mediatypes. Otherwise should update adapter code to use  app mediatypes